config/initializers/devise_token_auth.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+DeviseTokenAuth.setup do |config|
```

Comment:
this is not the correct configuration for devise token auth, take a look at the cooper documentation